api_tests/lib_sdk/actor_test.ts

```diff
@@ -1,6 +1,41 @@
 import { Actors } from "./actors";
 import { createActors } from "./create_actors";
 import { timeout } from "./utils";
+import { Actor } from "./actors/actor";
+import { createActor } from "./create_actor";
+/*
+ * Instantiates a new e2e test based on three actors
+ *
+ * This test function will take care of instantiating the actors and tearing them down again
+ * after the test, regardless if the test succeeded or failed.
+ */
+export function threeActorTest(
```

Comment:
No motivation to reduce the duplication between one, two and three actor test? :D